### PR TITLE
Convert Power Rail Track counters to be Monotonic

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

**Review by sonakshisaxena1 — APPROVED** [review]

**Comment by sonakshisaxena1** [comment]
This change converts the Power Rail Track counters to be Monotonic, thus correctly depicting the graph and the corresponding values in the form of delta values between timestamps.

Before: 
Screenshot 2022-08-08 at 4 22 42 pm

Now:
Screenshot 2022-08-08 at 4 24 04 pm

File: gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java [hunk 1]
```diff
@@ -27,27 +27,44 @@
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.gapid.models.Perfetto;
 import com.google.gapid.perfetto.Unit;
+import com.google.gapid.perfetto.models.QueryEngine.Row;
```
Comment (kocdemir) [inline gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java:30]: there are now mixed usages of 'Row' and 'QueryEngine.Row' in this file, could be better to keep using QueryEngine.Row as 'Row' is quite ambiguous.

**Review by kocdemir — APPROVED** [review]
lgtm, some minor changes can be done before or after submitting though, added comments at the lines.

File: gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java [hunk 2]
```diff
@@ -250,14 +271,69 @@ public static Type of(String string) {
...
+  }
+
+  private static class Stats {
+    public static final Stats ZERO = new Stats(0, 0, 0, 0, 0, 0, 0);
+
+    public final long count;
+    public final double stdMin;
+    public final double stdMax;
+    public final double stdAvg;
+    public final double monMin;
```
[the remark this answers is not in the document]
Comment (sonakshisaxena1) [reply gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java]: Makes sense. Updated.

File: gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java (hunk 1 above)
Comment (sonakshisaxena1) [reply gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java:30]: SGTM. Modified.

**Pull request merged**